### DISCUSSION TITLE
请问为什么文件下载后，里面的html文件打不开

### DISCUSSION THREAD

**Discussion by LazyKingOfQingQingGrassland** [byoungd/up#119] [Q&A]
![Uploading 微信图片_20230908005543.png…]()

**Comment by LazyKingOfQingQingGrassland** [comment]
![微信图片_20230908005543](https://github.com/byoungd/English-level-up-tips/assets/144299047/53fda238-cba2-4ca5-b57b-e824e046feca)

**Reply by byoungd** [reply]
因为这个html是为了GitHub Pages创建了，本地打开可能无法正常工作。

**Comment by LazyKingOfQingQingGrassland** [comment]
所以请问这个html文件不是这个项目运行，我作为操作方需要的打开的东西对吧。这个项目主要就是那几篇MD文件对吧


Fanson
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "byoungd/English-level-up-tips"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月8日(星期五) 中午11:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [byoungd/English-level-up-tips] 请问为什么文件下载后，里面的html文件打不开 (Discussion #119)





 
因为这个html是为了GitHub Pages创建了，本地打开可能无法正常工作。
